article/helpers.py

```diff
@@ -39,15 +39,15 @@ def count_average_word_number_in_lines_list(lines_list, word_length=5):
     return total_words
 
 
-def time_to_read_in_minutes(article):
+def time_to_read_in_seconds(article):
     """Return time to read in minutes give an Article object."""
     html = markdown_to_html(article.markdown_file_path)
     lines_list = lines_list_from_html(html)
     filtered_lines_list = filter_lines(lines_list)
     total_words_count = count_average_word_number_in_lines_list(
         filtered_lines_list
     )
-    return round(total_words_count / WORDS_PER_MINUTE, 2)
+    return round(total_words_count / WORDS_PER_SECOND, 0)
```

Comment:
The `ndigits` default is `0`, we can omit it.

You may want to change it [here](https://github.com/uktrade/directory-ui-export-readiness/blob/master/article/structure.py#L22) as well

Reply:
Probably best to remove the rounding and leave it for the display layer